adapters/smartadserver/smartadservertest/supplemental/request-site-recreated.json

```diff
@@ -27,7 +27,7 @@
   "httpCalls": [
     {
       "expectedRequest": {
-        "uri": "https://ssb.smartadserver.com/api/bid?callerId=5",
+        "uri": "https://ssb.smartadserver.com/api/bid?callerId=50",
```

Comment:
are these changes necessary ?

Reply:
yes, this was explicitly requested by smart adserver.